cypress/release.ts

```diff
@@ -4,17 +4,12 @@ describe(
   'Release',
   {
     pageLoadTimeout: ONE_MINUTE,
-    retries: 30,
+    retries: 60,
   },
   () => {
     it('loads swap page', () => {
       // We *must* wait in order to space out the retry attempts.
-      cy.wait(ONE_MINUTE)
-        .visit('/', {
```

Comment:
are these params defaulted or something?

Reply:
They're defaulted. I thought that changing the defaults would force retries, but I think it instead made retries occur without so much wait time between them, so I'm putting them back to defaults.

https://docs.cypress.io/api/commands/visit#Arguments